### ISSUE TITLE
Final Report Peer Review - rjb392

### ISSUE THREAD

**Issue by TheButlah** [fjt37/ml-big-data-project#15]
Things I liked:
- The rigor of the techniques you used was high (for example, you used cross-validation, train-test-validation splits).
- You explored most if not all of the models we learned about in class by looking at many different regularizers and loss functions.
- You analyzed the features to determine which were correlated and it helped reduce the number of features to allow for a simpler model
- You provided a good argument for how you chose your hyperparameters such as the historical time window to look at

Things I didn't like:
- You probably could have worked a bit more on thinking about a better output space. For example, why not let the output space be the change in price rather than the absolute price? I believe that this would help the model fit better. Maybe I'm totally wrong and thats fine but I think it would have been good to look into.
- The way you plot your price prediction graph seems misleading. You likely plotted the predicted price at every point, but its somewhat misleading because at each point it gets the correct values from the last several days, even though your predictions from those days were different from the true values. This means that the predictions will closely follow the true values just because there is only so much deviation the market will make in a day. The errors aren't cumulative since they will be reset at each time step. It would be more interesting to see what the model predicts if it has to predict several days into the future without getting told what the real price was. However, I don't think this is a big problem, its just a singular gripe I have with your prediction graphs.
- The feature for news events doesn't really solve the original issue about the volatility of the price, even if it did contribute to the model accuracy. If there is a news event the previous day and the model gets that as a feature, its effects would already have been reflected in the price, so its too little too late. There isn't really a good way to fix this and I think that it is indicative of the difficult nature of modeling bitcoin.

Overall I liked your project and think you did a good job. Congratulations!
